packages/ckeditor5-language/src/language.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module language/language
+ */
+
+import { Plugin } from 'ckeditor5/src/core';
+
+import LanguageEditing from './languageediting';
+import LanguageUI from './languageui';
+
+/**
+ * The language plugin.
+ *
+ * For more information about this feature check the {@glink api/language package page}.
+ *
+ * This is a "glue" plugin which loads the {@link module:language/languageediting~LanguageEditing language editing feature}
+ * and {@link module:language/languageui~LanguageUI language UI feature}.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class Language extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ LanguageEditing, LanguageUI ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'Language';
+	}
+}
+
+/**
+ * The configuration of the language feature. Introduced by the {@link module:language/languageediting~LanguageEditing} feature.
+ *
+ * Read more in {@link module:language/language~LanguageConfig}.
+ *
+ * @member {module:language/language~LanguageConfig} module:core/editor/editorconfig~EditorConfig#language
+ */
+
+/**
+ * The configuration of the language feature.
+ * The option is used by the {@link module:language/languageediting~LanguageEditing} feature.
+ *
+ *		ClassicEditor
+ *			.create( {
+ * 				language: ... // Language feature config.
+ *			} )
+ *			.then( ... )
+ *			.catch( ... );
+ *
+ * See {@link module:core/editor/editorconfig~EditorConfig all editor options}.
+ *
+ * @interface LanguageConfig
+ */
+
+/**
+ * The available language options.
+ *
+ * The default value is:
```

Comment:
Is the default from CKE4? It's quite useless, but I suppose it's on purpose – it ensures that the feature works by default, but requires to be reconfigured when integrated for real.

Thus, we don't need to have translations for these language titles. Note: in many other features such strings are covered by language files and we have automatic translations for them. But here, it makes no sense.

This is something that may not be obvious to others (that we don't translate them on purpose), so it'd be good to have a comment about that in the code (next to the default value?). It may be hard to spot if you don't know that you're looking for it, but it's better than nothing.

Reply:
> Is the default from CKE4? It's quite useless, but I suppose it's on purpose – it ensures that the feature works by default, but requires to be reconfigured when integrated for real.

Yes, and yes :)  that's the purpose of this default configuration in CKE4.